packages/css/src/switch/switch.scss

```diff
@@ -0,0 +1,68 @@
+.amsterdam-switch {
+  display: inline-block;
+  font-family: var(--amsterdam-switch-font-family);
+  font-size: var(--amsterdam-switch-font-size-narrow);
+  line-height: var(--amsterdam-switch-line-height);
+  outline-offset: var(--amsterdam-switch-outline-offset);
+  overflow: visible;
+  position: relative;
+}
+
+.amsterdam-switch__input {
+  appearance: none;
+  margin-block: 0;
+  margin-inline: 0;
+  opacity: 0%; /* to hide native input field in older iOS */
+}
+
+.amsterdam-switch__toggle {
+  background-color: var(--amsterdam-switch-background-color);
+  border-radius: 50px;
+  box-sizing: border-box;
+  display: inline-block;
+  padding-block: 1px;
+  padding-inline: 1px;
+  transition: background-color 0.2s ease-in-out;
+  width: var(--amsterdam-switch-toggle-width);
+}
+
+.amsterdam-switch__toggle-inner {
+  background-color: var(--amsterdam-switch-toggle-background-color);
+  border-radius: 50%;
+  display: block;
+  height: var(--amsterdam-switch-toggle-inner-height);
```

Comment:
Do we want to hook this up to our text levels as well? So the switch will be as tall as the line height of a text level?